src/Manager/ImageManager.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Manager;
+
+use App\Entity\Image;
+
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class ImageManager
+{
+    public function __construct()
+    {
+    }
+
+    public function uploadService()
+    {
+        $image = new Image;
+
+        $file = $image->getImage();
+        $id = md5(uniqid()).'.'.$file->guessExtension();
+        $file->move($this->getParameter('images_directory'), $id);
+
+        return $this->redirect($this->generateUrl('download', array('id' => $id)));
+    }
+
+    public function downloadService($route, $id)
```

Comment:
pas besoin de passer $route en paramètre, tu l'injecte déjà dans ton service.yaml : https://github.com/Ozyhcs/SymfonyAPI/blob/dev/config/services.yaml#L18
Du coup dans ce fichier tu peux faire ajouter avant les méthodes un `protected $imagesDirectory;`
Et ton construct devient : 
public function __construct(string $imageDirectory){
$this->imageDirectory = $imageDirectory;
}
Enfin dans tes méthodes tu peux directement utiliser $this->imageDirectory au lieu de passer le $route en paramètre